src/kubernetes-tool/utils/helm_parts/functions/if.ts

```diff
@@ -96,7 +91,8 @@ const processCondition = (parser: DocumentParser, args: (string | Quote)[]): boo
                 // This *should* be the condition.
                 return Boolean(parser.helmdef2object(operator))
             } else {
-                throw new Error(`"${operator}" - Invalid operator!`)
+                // Returns false if it doesn't understand.
+                return false
```

Comment:
What does this look like to the user?

Reply:
Just picks the else statement. This is fine because it is something that we do not have access to client-side in that situation.